wagtail/api/v2/filters.py

```diff
@@ -78,49 +79,26 @@ def filter_queryset(self, request, queryset, view):
         And random ordering
         Eg: ?order=random
         """
-        if "order" in request.GET:
-            order_by_list = request.GET["order"].split(",")
+        order_param = request.GET.get("order")
+        if not order_param:
+            return queryset
 
-            # Random ordering
-            if "random" in order_by_list:
-                if len(order_by_list) > 1:
-                    raise BadRequestError(
-                        "random ordering cannot be combined with other fields"
-                    )
-                # Prevent ordering by random with offset
-                if "offset" in request.GET:
-                    raise BadRequestError(
-                        "random ordering with offset is not supported"
-                    )
-
-                return queryset.order_by("?")
-
-            order_by_fields = []
-            for order_by in order_by_list:
-                # Check if reverse ordering is set
-                if order_by.startswith("-"):
-                    reverse_order = True
-                    order_by = order_by[1:]
-                else:
-                    reverse_order = False
-
-                # Add ordering
-                if order_by in view.get_available_fields(queryset.model):
```

Comment:
I notice the new code is no longer validating the list against `view.get_available_fields`. I think it's important to keep this - if there are private fields that exist on the model but aren't exposed in the API, being able to order by those fields could potentially leak information.

(Also, we should probably be passing [the `db_fields_only` flag](https://github.com/wagtail/wagtail/blob/47fc2008640ece76be97f3d03da224b903a99047/wagtail/api/v2/views.py#L170-L178) here...)